content/docs/06-integrations/00-prometheus-operator.md

```diff
@@ -36,6 +36,165 @@ We recommend you use version v44.3.x or greater moving forward for a simplified
 ## Versions Supported
 <Tabs>
 
+<Tabs.TabPane tab="45.4.x" key="45.4.x">
+
+## Prerequisites
+
+* Kubernetes v1.16 or greater.
+
+
+* The minimum required size for the Prometheus server is 4 CPU, 8 GB Memory, and 10 GB Storage. We recommend the monitoring stack have 1.5x to 2x the minimum required size:
+
+  Recommended size:
+  - 8 CPU 
+  - 16 GB Memory
+  - 20 GB Storage. 
+ 
+ 
+  As new clusters with the Prometheus agent are added to your environment, review the resource utilization and consider increasing resources if needed. As the Prometheus documentation recommends, each additional agent requires the following resources from the monitoring stack:
+
+  Each added agent:
+    - 0.1 CPU
+    - 250 MiB Memory
+    - 1 GB Storage.

```

Comment:
```suggestion
    - 1 GB Storage

```

Reply:
Several of our docs say "GiB". We don't have a style for that yet and should probably come up with one since it's an inconsistency in our docs.